.github/workflows/integration-tests-vm-type.yml

```diff
@@ -107,7 +107,9 @@ jobs:
 
       - name: Teardown VMs
         if: always()
-        run: make -C "${{ github.workspace }}/ansible" destroy-vms
+        run: |
+          make -C "${{ github.workspace }}/ansible" list-inventory
```

Comment:
Is this line for debugging? Do we intend to keep it?

Reply:
ahh yeah it was for debugging. I don't think it's strictly necessary, though hiding it behind GHA debug flag might be pretty useful:

```
        if [[ "${RUNNER_DEBUG}" == "1" ]]; then
          make -C "${{ github.workspace }}/ansible" list-inventory
        fi
```

WDYT?